docs/pages/blog/premium-plan-release.md

```diff
@@ -78,7 +78,7 @@ We’d also like to keep it within reach of individuals and small teams, while s
 Here is a list of everything that has changed:
 
 :::warning
-⚠️ Please note: The following **pricing** changes apply only to **new customers**. Existing customers are grandfathered, they can renew their license under the [previous terms](https://mui.com/store/legal/mui-x-eula-2022-05-08/) once. Their last renewal can have a support duration of up to 5 years.
+⚠️ Please note: The following **pricing** changes apply only to **new customers**. Existing customers are grandfathered, they can renew their license under the [previous terms](https://mui.com/legal/mui-x-eula-2022-05-08/) once. Their last renewal can have a support duration of up to 5 years.
```

Comment:
```suggestion
⚠️ Please note: The following **pricing** changes apply only to **new customers**. Existing customers are grandfathered in, they can renew their license under the [previous terms](https://mui.com/legal/mui-x-eula-2022-05-08/).
```

This was an idea you floated, but it was never agreed. To be removed ASAP.

Reply:
Interesting, I think that we should consider this for a follow-up. This PR is about creating a new legal scope, so it's off-topic.

I feel that almost nobody renews, so no objections to moving in the direction you are pushing here. Personally, I would be in favor of keeping the current tradeoffs so we can more quickly "remove pricing debt".